rust_src/src/eval_macros.rs

```diff
@@ -283,3 +283,28 @@ macro_rules! per_buffer_var_idx {
         (unsafe { crate::remacs_sys::buffer_local_flags.$field }).as_natnum_or_error() as usize
     };
 }
+
+#[macro_export]
+#[allow(unused_macros)]
+macro_rules! local_unibyte_string {
+    ($name: ident, $string: expr) => {
+        let mut bytes: Vec<u8> = ($string).bytes().collect();
```

Comment:
Could this have been `$string.as_str().as_bytes_mut()`?

Reply:
That would mean making `$string` mutable.

Reply:
Is that a significant concern here?